doc/vision_mission_strategy.md

```diff
@@ -0,0 +1,39 @@
+## Network-NG (Vision, Mision and Strategy)
+  
+It [has been decided](https://github.com/yast/yast-network/blob/master/doc/network-ng-kickstart.md#refactoring-yast-network---kickstart-meeting) that now was a perfect moment to start with the **YaST Network** refactoring. 
+  
+Some of the network areas are quite big and will need a large effort to be completely rewritten.  Thus, it is very important to define as a group effort a clear vision and develop a clear mision to make the refactor achievable as well to determine the strategy we will follow during the process. 
```

Comment:
*clear* is repeated (use a synonym or just drop one of them).

Reply:
I would drop *as a group effort*. Moreover, I would simplify this sentence, as it might be hard to read. What about something like "Thus, it is critical to set a common vision and define a clear strategy".